packages/fiori/src/themes/TimelineItem.css

```diff
@@ -231,7 +231,6 @@
 	font-weight: 400;
 	font-size: var(--sapFontSmallSize);
 	padding-top: var(--_ui5_TimelineItem_bubble_content_subtitle_padding_top);
-	white-space: nowrap;
 	overflow: hidden;
```

Comment:
1. Looking at the these, maybe the goal was to truncate text if it overflows 
```css
white-space: nowrap;
overflow: hidden;
text-overflow: ellipsis;
```

2. If you remove `white-space: nowrap;`, then `text-overflow: ellipsis` and `overflow: hidden;` do not make sense - they should be also  removed

So, first we need to understand if the text should truncate or wrap.  In case it needs to truncate, different fix is required. If needs to wrap, then you need to remove more styles as obsolete.

Reply:
It is not clear according to the UX guidelines if the text should wrap or truncate.
There is an example there showcasing a text wrapping for three lines and the fourth line gets truncated.
We could use the openui5 timeline control for reference. There all the texts (Title, sub-title, text) do wrap and never truncate.

Reply:
Ok, then let's do the wrapping, but clean-up the styles related to the truncation as mentioned in my comment.